zipkin-autoconfigure/collector-scribe/README.md

```diff
@@ -13,14 +13,14 @@ JRE 8 is required to run Zipkin server.
 Fetch the latest released
 [executable jar for Zipkin server](https://search.maven.org/remote_content?g=io.zipkin.java&a=zipkin-server&v=LATEST&c=exec)
 and
-[autoconfigure module jar for the scribe collector](https://search.maven.org/remote_content?g=io.zipkin.java&a=zipkin-autoconfigure-collector-scribe&v=LATEST&c=module).
+[autoconfigure module jar for the scribe collector](https://search.maven.org/remote_content?g=io.zipkin.zipkin2&a=zipkin-autoconfigure-collector-scribe&v=LATEST&c=module).
```

Comment:
```suggestion
[autoconfigure module jar for the scribe collector](https://search.maven.org/remote_content?g=org.apache.zipkin.zipkin2&a=zipkin-autoconfigure-collector-scribe&v=LATEST&c=module).
```

Reply:
there's a separate task for this, as this isn't published yet. #2513